docker-compose.yml

```diff
@@ -153,19 +153,22 @@ services:
     environment:
       - SERVICES=firehose,kinesis,s3,iam,sts,secretsmanager
 
-  # s3 compatible object storage (file carving backend)
-  minio:
-    image: quay.io/minio/minio
-    command: server /data --console-address ":9001"
+  # s3 compatible object storage (file carving/software installers)
+  s3:
+    image: rustfs/rustfs:latest
```

Comment:
Maybe a-ok given this is for dev/testing but we could also pin the version here?

Reply:
Fair enough on being consistent between pinned and latest since that gives us a lot of representative testing on fleetctl preview for free.